### PR TITLE
docs(update)

### PR DESCRIPTION
**Pull request by lucasrodes**
- Improvements on organisation
- Updates to keep the documentation up to date (walden, dataset examples, etc.)
- Minor style adjustmenets

### CODE REVIEW

File: docs/adding-data/manual.md [hunk 1]
```diff
@@ -32,63 +35,87 @@ You will also need to choose a short name for the dataset itself (e.g. `populati
...
+```
+
+### Create Meadow step
 
-    These instructions use `walden`, our old system for snapshotting data. They need to be updated to use `snapshots` instead.
+#### Create a new `meadow` step file
 
-0. Activate the virtual environment (running `poetry shell`).
+- Path of the step should be similar to `etl/steps/data/meadow/[namespace]/[version]/[dataset_short_name].py`.
+- The step must contain a `run(dest_dir)` function that loads data from the `walden` bucket in S3 and creates a dataset
```
Comment (pabloarosado) [inline docs/adding-data/manual.md:63]: Why not updating it to snapshot?

File: docs/adding-data/manual.md [hunk 2]
```diff
@@ -32,63 +35,87 @@ You will also need to choose a short name for the dataset itself (e.g. `populati
...
+- Path of the step should be similar to `etl/steps/data/garden/[namespace]/[version]/[dataset_short_name].py`.
+- The step must contain a `run(dest_dir)` function that loads data from the last `meadow` step, processes the data and
+creates a dataset with one or more tables and the necessary metadata.
+- Country names must be harmonized (for which the [harmonize](harmonization.md) tool of `etl` can be used).
+- Add plenty of assertions and sanity checks to the step (if possible, compare the data with its previous version and
+check for abrupt changes).
+- Run `make format && make test` to ensure that the step runs well and is well formatted.
 
-5. **Create a pull request** to merge the new branch with the master branch in `walden`. When getting started with the `etl` you should request a code review from a more experienced `etl` user.
+#### Add the new garden step to the dag**, including its dependencies.
```
Comment (pabloarosado) [inline docs/adding-data/manual.md:88]: Why the **?

File: docs/adding-data/manual.md [hunk 3]
```diff
@@ -32,63 +35,87 @@ You will also need to choose a short name for the dataset itself (e.g. `populati
...
-    Add `--grapher` flags to `etl` command to upsert data into grapher database.
-    To test the step, you can run it on the grapher `staging` database, or using
-    [a local grapher](https://github.com/owid/owid-grapher/blob/master/docs/docker-compose-mysql.md).
+To test the step, you can run it on the grapher `staging` database, or using
+[a local grapher](https://github.com/owid/owid-grapher/blob/master/docs/docker-compose-mysql.md).
 
-14. **Create a pull request** to merge the new branch with the master branch in `etl`.
-    At this point, some further editing of the step files may be required before merging the branch with master.
\ No newline at end of file
+#### Create a pull request** to merge the new branch with the master branch in `etl`.
```
Comment (pabloarosado) [inline docs/adding-data/manual.md:120]: Why the **?

File: docs/adding-data/walkthrough.md [hunk 4]
```diff
@@ -1,41 +1,49 @@
...
-Before you begin, make sure you've set up the ETL as described in [Getting Started](../getting-started/index.md).
-
-## Decide on names
+The walkthrough is an interactive web UI for setting up the different ETL steps. It creates the base files to help you
+create the recipe for a dataset. It currently supports the following stages:
 
-The ETL uses a naming convention to identify datasets. To add a dataset, you will need to choose a short name for the data provider (e.g. `un`, `who`), which will serve as the namespace to add it to.
+| Option      | Description                          |
+| ----------- | ------------------------------------ |
+| `walden`       | Create a Walden step: Insert upstream data to our catalog. |
```
Comment (pabloarosado) [inline docs/adding-data/walkthrough.md:12]: Mention that this is deprecated.

File: docs/adding-data/walkthrough.md [hunk 5]
```diff
@@ -1,41 +1,49 @@
...
+| `snapshot`       | Create a Walden step: Insert upstream data to our catalog. |
+| `meadow`       | Create a Meadow step: Format data. |
+| `garden`       | Create a Garden step: Harmonize and process data. |
+| `grapher`       | Create a Grapher step: Transform data to be Grapher-ready. |
+| `explorers`       | Create an Explorer step: Transform data to be Explorer-ready. |
```
Comment (pabloarosado) [inline docs/adding-data/walkthrough.md:17]: Mention the `charts` option?

File: docs/adding-data/manual.md (hunk 2 above)
Comment (lucasrodes) [reply docs/adding-data/manual.md:88]: typo

File: docs/adding-data/manual.md (hunk 3 above)
Comment (lucasrodes) [reply docs/adding-data/manual.md:120]: typo

**Review by Marigold — APPROVED** [review]
I'm going to cry; this is so nice!

**Pull request merged**